exercises/exercise-three/form/Signup_dom.js

```diff
@@ -0,0 +1,89 @@
+// define elements
+const form = document.querySelector('.signup-form');
+const email = document.querySelector('#email');
+const username = document.querySelector('#user-name');
+const password = document.querySelector('#password');
+const confirmPassword = document.querySelector('#password-confirm');
+
+// Validate and display error messages by creating text and assigning to it's P tags
+const checkError = (element, message) => {
+  const showMessage = element.parentElement.querySelector('.form-message');
+
+  showMessage.innerText = message;
+};
+
+// Show nothing when value is valid
+const checkSuccess = (element) => {
+  const showMessage = element.parentElement.querySelector('.form-message');
+
+  showMessage.innerText = '';
+};
+
+// Validate email address with regular expressions (Stack Over Flow)
+const validateEmail = (input) => {
+  const regExp = /^(([^<>()[\]\.,;:\s@\"]+(\.[^<>()[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
+  return regExp.test(input);
+};
+
+// Function display inputs to the screen
+const displayInfo = () => {
+  let result = '';
+
+  for (i = 0; i < (form.length - 1); i++) {
```

Comment:
Don't use magick variable name like i, j, k etc ...
Should use index, jindex or order ...

Reply:
form.length - 1 . SHould be not exist in here.

let total = form.length-1

for (let index = 0; index < total; index++) {